scripts/install/install-pocs.sh

```diff
@@ -50,15 +50,15 @@ usage() {
  If in DEVELOPER mode, the following options are also available:
   USER      The PANUSER environment variable, defaults to current user (i.e. PANUSER=$USER).
   PANDIR    Default install directory, defaults to PANDIR=${PANDIR}. Saved as PANDIR
-            environment variable.
+            environment variable. Test $LOGFILE.
```

Comment:
There is a line somewhere below that prints all the options where this could be added. But shouldn't be needed here.

```suggestion
            environment variable.
```